pages/overview/custom-pages.mdx

```diff
@@ -0,0 +1,195 @@
+import { Callout } from "nextra/components";
```

Comment:
This page I would like to have it after _authentication_ 💅🏽 

Once you have set up an authentication method (OAuth, magic links or credentials) the next thing to have a "completed setup" is to hook the session provider and consume the session to display the user logged-in status. 

I'm thinking what is the best way to communicate this 👀

Reply:
Yeah so that makes sense to me. But in general I was a bit confused about this page, it sounded like it was supposed to be about custom pages but then it also mentioned session / user-managament a lot.

Maybe we keep this "Custom Pages" where it is and add back a new "Managing Session" (/User) for what you intended back under "Authentication"? Sound good?

Reply:
Added back a page for this [here](https://github.com/ubbe-xyz/nextra-docs/pull/2/commits/86cf639f06f9caa44740c089b753c2d6dc88a56c) :+1: